GETTING_STARTED.md

```diff
@@ -29,7 +36,7 @@ Then you can setup the initial admin account and initialise the database into yo
 You can now run the server - note that we provide all the options on the cli, but this pattern
 may change in the future.
 
-    docker run -p 443:8443 -v /Users/william/development/rsidm/insecure:/data firstyear/kanidmd:latest /home/kanidm/target/release/kanidmd server -D /data/kanidm.db -C /data/ca.pem -c /data/cert.pem -k /data/key.pem --bindaddr 0.0.0.0:8443 --domain localhost
+    docker run -p 8443:8443 -v /Users/william/development/rsidm/insecure:/data firstyear/kanidmd:latest /home/kanidm/target/release/kanidmd server -D /data/kanidm.db -C /data/ca.pem -c /data/cert.pem -k /data/key.pem --bindaddr 0.0.0.0:8443 --domain localhost
```

Comment:
i guess you technically don't need _any_ of the arguments on the end of this kanidmd command, especially since they're in the dockerfile?

alternatively having sane defaults for kanidmd and only specifying overrides might be better, idk.

Reply:
See belowe - it's the -d parameter.

Reply:
Sorry, -d == --domain